os/linux/gbmbufferhandler.h

```diff
@@ -25,6 +25,69 @@ namespace hwcomposer {
 
 class GpuDevice;
 
+#ifndef USE_MINIGBM
```

Comment:
Can we move this into "gbmutils.h" ? can have it outside USE_MINIGBM and include it tests only if needed